### DISCUSSION TITLE
AppFlow On HELLOTUX? #shirt #hoodie #polo

### DISCUSSION THREAD

**Discussion by eilissadh** [AppFlowy-IO/AppFlowy#1367] [Ideas]
What do you think about asking [Gabor](mailto:gabor@hellotux.com) from [HELLOTUX](https://www.hellotux.com/) to add AppFlowy wear to his shop?
His stuff has an awesome quality and he is a true open source lover.

**Comment by yllenfer** [comment]
I'd love to wear some AppFlowy shirts!
